backend/benefit/applications/api/v1/serializers.py

```diff
@@ -57,7 +57,9 @@ def __call__(self, value, serializer_field):
             ):
                 raise serializers.ValidationError(
                     _(
-                        f"Application state transition not allowed: {application.status} to {value}"
+                        "Application state transition not allowed: {status} to {value}".format(
+                            status=application.status, value=value
+                        )
```

Comment:
When discussing this ticket I didn't realize that f strings are not supported by gettext... The issue is more complex than I thought 

Question about format(): does it work here? https://docs.djangoproject.com/en/3.2/topics/i18n/translation/ says that format() doesn't work on lazy translation objects and format_lazy should be used instead? Or should we do
"from django.utils.translation import gettext as _", as I think it's ok to translate the string immediately when raising ValidationErrors?

Reply:
Aha, good catch, I think we should use the format_lazy instead of using gettext. There are someplace where we have to use gettext_lazy so it would end up us using two different gettext functions. I'll update the PR